dandalandone/POC-EyCommerce
# [Frontend]: Cart table component(see notes)

## Ticket/s

### [Cart table component](https://trello.com/c/wumTBHXD/84-display-cart-items-component)
  _Other tickets with changes:_
  - [Cart drop down component in header](https://trello.com/c/u7CDrYvL/52-cart-dropdown-view)
  - [Quantity component](https://trello.com/c/L21PspvK/99-increase-decrease-item-quantity-component)

## Changes

- Create cart table in temporary cart view
- Cart row component for each item
- Utilize quantity +/- component
- Responsize cart using defined breakpoints
Other:
- Set default breakpoints in theme
- Add import for bold font
- Add responsiveness to quantity +/- component

## Screenshot

![cartpageweb](https://github.com/dandalandone/POC-EyCommerce/assets/132859942/5476cd15-d16a-4b57-85a8-288109ff57b1)
![image](https://github.com/dandalandone/POC-EyCommerce/assets/132859942/46b5f84a-9e07-4b44-8667-c79e364a29e2)


## How to test?

- checkout to branch
- pnpm run dev
- http://localhost:5173/cart
_notes: 
-also check on cart dropdown for the updated quantity component
-confirm if the small device design is also approved_
